src/main/java/org/skriptlang/skript/bukkit/fishing/elements/EffPullHookedEntity.java

```diff
@@ -0,0 +1,50 @@
+package org.skriptlang.skript.bukkit.fishing.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerFishEvent;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Pull Hooked Entity")
+@Description("Pull the hooked entity to the player.")
+@Examples({
+	"on fishing state of caught entity:",
+		"\tpull hooked entity"
+})
+@Events("Fishing")
+@Since("INSERT VERSION")
+public class EffPullHookedEntity extends Effect {
+
+	static {
+		Skript.registerEffect(EffPullHookedEntity.class, "(reel|pull) [in] hook[ed] entity");
```

Comment:
i don't really like in being optional tbh, it feels a bit strange
i think either `pull in hooked entity` or `pull hooked entity in`